openassessment/xblock/test/test_openassessment.py

```diff
@@ -202,6 +207,33 @@ def test_ora_blocks_listing_view(self, xblock):
         items = json.loads(scripts[0].text)
         self.assertEqual(items, defined_ora_items)
 
+    @scenario('data/basic_scenario.xml')
+    @ddt.data(
+        *list(itertools.product([True, False]))
+    )
+    @ddt.unpack
```

Comment:
I might be missing something, but this looks like it's just a complicated way of doing
```suggestion
    @ddt.data(False, True)
```